README.md

```diff
@@ -1,60 +1,13 @@
-
-## Deployment packages
-To generate local deployment packages:
-
-  1. From the [project release page](https://github.com/fortinet/fortigate-autoscale/releases), download the source code (.zip or .tar.gz) for the latest version.
-  2. Extract the source code.
-  3. Run `npm run build` at the project root directory.
-
-Deployment packages as well as source code will be available in the **dist** directory.
-
-| Package Name                                      | Description                                                            |
-| ------------------------------------------------- | ---------------------------------------------------------------------- |
-| fortigate-autoscale.zip                           | Source code for the entire project.                                    |
-| fortigate-autoscale-azure-funcapp.zip             | Source code for the FortiGate Autoscale handler - Azure function.      |
-| fortigate-autoscale-azure-template-deployment.zip | Azure template. Use this to deploy the solution on the Azure platform. |
-
-## Deployment guide
-A deployment guide is available from the Fortinet Document Library:
-
-  + [ FortiGate / FortiOS 6.4 / Deploying auto scaling on Azure](https://docs.fortinet.com/vm/azure/fortigate/6.4/azure-cookbook/6.4.0/161167/deploying-auto-scaling-on-azure)
-
-# Launch a demo
-FortiGate Autoscale for Azure requires the deployment of two templates in sequence into the same resource group.
-| Step | Action |
-|------|---------------|
-| 1 | Deploy the stage 1 template using the [deployment parameters file](https://github.com/fortinet/fortigate-autoscale/blob/main/azure_template_deployment/templates/deploy_fortigate_autoscale.hybrid_licensing.params.json).<br><a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Ffortinet%2Ffortigate-autoscale%2Fmaster%2Fazure_template_deployment%2Ftemplates%2Fdeploy_fortigate_autoscale.hybrid_licensing.json" target="_blank"><img src="http://azuredeploy.net/deploybutton.png"/></a> |
-| 2 | Wait for the stage 1 deployment to complete. |
-| 3 | Deploy the stage 2 template using the same [deployment parameters file](https://github.com/fortinet/fortigate-autoscale/blob/main/azure_template_deployment/templates/deploy_fortigate_autoscale.hybrid_licensing.params.json) with the same parameter values used in step 1.<br><a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Ffortinet%2Ffortigate-autoscale%2Fmaster%2Fazure_template_deployment%2Ftemplates%2Fdeploy_fortigate_autoscale.hybrid_licensing.json" target="_blank"><img src="http://azuredeploy.net/deploybutton.png"/></a> |
-
-Refer to the [documentation](https://docs.fortinet.com/vm/azure/fortigate/6.4/azure-cookbook/6.4.0/161167/deploying-auto-scaling-on-azure) for descriptions of the deployment parameters file.
-
-## Project development history
```

Comment:
Related to the previous question. If we are including the link to the last release in this project, we should also include these.
If you agree, I suggest changing the title above to "Project history". Please send back to me for updates.

Reply:
Users / readers can switch to the 3.1.2 tag branch to see the history whenver they want. Our goal here is to eliminate almost all contents but just two pieces of index for reference, one for the last full version of this repo and the other is the starting point of the new repo.

Reply:
btw, everything in the main branch except for the README will be gone at the end.

Reply:
I understand now. Thanks for the explanation.